ComfortZoneApp/app/src/main/java/com/example/comfortzone/models/WeatherData.java

```diff
@@ -4,25 +4,27 @@
 
 public class WeatherData {
 
-
     @SerializedName("name")
     private String city;
-    @SerializedName("feels_like")
-    private int temp;
+
+    @SerializedName("main")
+    private TempData tempData;
 
     public String getCity() {
         return city;
     }
-
-    public void setCity(String city) {
-        this.city = city;
+    
+    public Object getTempData() {
```

Comment:
qq: does the return type have to be `Object`? couldn't you use `TempData` to avoid casting from the callsite?